folding-schemes/examples/utils.rs

```diff
@@ -0,0 +1,48 @@
+#![allow(non_snake_case)]
+#![allow(non_upper_case_globals)]
+#![allow(non_camel_case_types)]
+#![allow(clippy::upper_case_acronyms)]
+#![allow(dead_code)]
+use ark_pallas::{constraints::GVar, Fr, Projective};
+use ark_vesta::{constraints::GVar as GVar2, Projective as Projective2};
+
+use folding_schemes::commitment::pedersen::Pedersen;
+use folding_schemes::folding::nova::{get_r1cs, ProverParams, VerifierParams};
+use folding_schemes::frontend::FCircuit;
+use folding_schemes::transcript::poseidon::poseidon_test_config;
+
+// This method computes the Prover & Verifier parameters for the example. For a real world use case
+// those parameters should be generated carefuly (both the PoseidonConfig and the PedersenParams)
+#[allow(clippy::type_complexity)]
+pub(crate) fn nova_setup<FC: FCircuit<Fr>>(
```

Comment:
I would use this name to emphasize that this should not be used in prod in any way.

```suggestion
pub(crate) fn test_nova_setup<FC: FCircuit<Fr>>(
```

Reply:
Adding the `test_` prefix makes it look as it is a test, but it is not a test but a function to be used in tests. It is in fact not a public function outside the crate, so users of the library should not have access to it.
But agree with you on emphasizing that this should not be used in production, added an extra 'warning' line in the comment of the function.

Reply:
Update: I had a second thought on it and changed my mind, even the function is not a test itself, I agree that is a priority to make it as more explicit as we can that is intended to be used only in tests. Updated here: https://github.com/privacy-scaling-explorations/folding-schemes/pull/73/commits/f64b13e05d87f2be7ef330230c7a6550a0558dad